<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCARD</class>
 <widget class="QDialog" name="CCARD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="QpbtnIsOk">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnClose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>78</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>63</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>有效期：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="QdateValidity"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>房间号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="QlineAddr">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>卡号：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>00：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="QlineCardNum">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="QcomCheck1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="QcomCheck2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="QcomCheck3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="QcheckBox1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="QcheckBox2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="QcheckBox3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
